src/layouts/EditHomepage.jsx

```diff
@@ -904,6 +899,16 @@ const EditHomepage = ({ match }) => {
     return false
   }
 
+  // NOTE: sectionType is one of `resources`, `infopic` or `infobar`
+  const onClick = (sectionType) => {
+    createHandler({
+      target: {
+        value: sectionType,
+        id: "section-new",
```

Comment:
Shouldn't the id be dynamic based on the sectionType?

Reply:
sadly this was the `id` used in the legacy code too ._. (see [here](https://github.com/isomerpages/isomercms-frontend/pull/1415/commits/0c02ca9b68fefa30a574dae88a5dd9862968e225#diff-a2ed50d7269adce5178a9e394aed981d74165d870a24d31393f4152af4521e68L35)) - the `createHandler` itself actually does all the logic iirc